doc/api/next_api_changes/deprecations/22134-OG.rst

```diff
@@ -0,0 +1,5 @@
+Modules ``tight_bbox``, ``tight_layout`` deprecated
+------------------------------------------------------------
```

Comment:
```suggestion
Modules ``tight_bbox`` and ``tight_layout``
-------------------------------------------
```

Reply:
Gah! That is what happens when you copy and paste too much and think too little... Thanks!